src/css/flags/ads.less

```diff
@@ -67,7 +67,7 @@
 }
 
 .jwplayer.jw-flag-ads-hide-controls {
```

Comment:
This will only work for ads that have their own controls. Linear video ads use jwplayer's controls, and would still display captions in this case. Please find a solution using the `jw-flag-ads` class.